nautobot/core/tests/test_tables.py

```diff
@@ -18,8 +18,18 @@ class TableTestCase(TestCase):
     def _validate_sorted_tree_queryset_same_with_table_queryset(self, queryset, table_class, field_name):
         with self.subTest(f"Assert sorting {table_class.__name__} on '{field_name}'"):
             table = table_class(queryset.with_tree_fields(), order_by=field_name)
-            table_queryset_data = table.data.data.values_list("pk", flat=True)
-            sorted_queryset = queryset.with_tree_fields().extra(order_by=[field_name]).values_list("pk", flat=True)
+            table_queryset_data = table.data.data.values_list(field_name, flat=True)
+            sorted_queryset = (
+                queryset.with_tree_fields().extra(order_by=[field_name]).values_list(field_name, flat=True)
+            )
```

Comment:
Note the change here from `values_list("pk")` to `values_list(field_name)`. I found that "before" this fix, the PK listing was consistently ordered even for records with identical values for a given `field_name` (e.g., multiple Locations with the same `parent` value), whereas "after" the fix, the queryset ordering did **not** consistently match the table ordering among such records. I think this is "acceptable" as we're specifying only a single-field ordering, with no explicit fallback field to order among the records with identical values for that field, and so updated the test accordingly. But I wanted to call attention to that in case there's disagreement on this point.